docs/prompt_engineering/how_to_guides/playground/testing_over_dataset.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 5
+---
+
+# Testing over a dataset
+
+It's often useful to test your prompts over a dataset to understand how your model will behave in different scenarios. The Playground allows you to easily test your prompts over any of your LangSmith datasets. 
+
+## Running your prompt
+
+1. Select the prompt you want to test. This can either be a saved or unsaved prompt. If you add multiple prompts to the playground, they will all be run over the dataset.
```

Comment:
id add a brief note on how to add another prompt (click +compare)

Reply:
like click compare if you want to compare the performance of two prompts over the dataset